lib/content-style/parser.rb

```diff
@@ -16,6 +16,28 @@ def parse(file_content)
         file_tree
       end
 
+      def get_erb_locations(file_content)
+        hotcop_lines = []
+        hotcop_identifiers = []
+        current_line_number = 1
+        s = StringScanner.new(file_content)
+        while (line_content = s.scan_until(/\n/))
+          if line_content =~ /HOTCOP START$/
+            identifier_line_number = current_line_number + 2
+            hotcop_lines.push(identifier_line_number)
```

Comment:
Since you're only ever using `.last` of this array, I'd just use `identifier_line_number` (or some other appropriately named variable) and check against that in the `if` statement below